.github/workflows/vanilla_workflow.yaml

```diff
@@ -37,7 +37,16 @@ jobs:
           REPLICATE_API_TOKEN: ${{ secrets.REPLICATE_API_TOKEN }}
         run: |
           notebooks=$(cat .github/notebook_lists/vanilla_notebooks.txt | tr '\n' ' ')  
-          for notebook in $notebooks; do  
-            echo "Executing $notebook"  
-            jupyter nbconvert --to notebook --execute --inplace $notebook  
+          for notebook in $notebooks; do 
+            echo "Building virtual env"  
+            rm -rf .venv
+            python -m venv .venv
```

Comment:
Same comment as other PR, 
You shouldn't need a virtual environment, but if you _really_ do, it should be created in it's own step.

Reply:
@rawkintrevo the purpose of this PR is to test each notebook in its own virtual env. How would you otherwise test each notebook with a clean environment to make sure it has taken care of all of its installs independently of the other notebooks in the cookbook?

Reply:
OK, I now understand what you're doing. The 'right' way is probably to just create a workflow for each notebook then. But I understand counter arguments against that. IF you're going to go forward with this method, I would at least delete the `venv` folder after deactivating the virtual environment.

Reply:
oh it does that at the beginning of the loop. Does `rm` fail if the directory isn't there (iirc you can put a flag on it so that it doesn't but I haven't done this for a while)

Reply:
As long as it's `rm -rf ...`, the `-f` will cause `rm` to silently ignore a missing file or directory.

Reply:
I'm seeing these with the discussion towards the end of this set of PRs.

I think these are a step forward, though maybe not the last step.  I've been pondering the suggestion to just have one workflow file per recipe.  It would solve some problems.  But having a 1:1 workflow to `.ipynb` file seems like an antipattern.

I'm inclined to pull all these in, since they are finding bugs. Can we do that and discuss at next meeting?